code/control/DeploynautLogFile.php

```diff
@@ -14,17 +15,29 @@ public function getLogFile() {
 		return $this->logFile;
 	}
 
+	/**
+	 * Write a message line into the log file.
+	 * @param string $message
+	 */
 	public function write($message) {
 		error_log('['.date('Y-m-d H:i:s').'] ' . $message .PHP_EOL, 3, $this->logFile);
 		@chmod($this->logFile, 0666);
 	}
 
+	/**
+	 * Does the log file exist?
+	 * @return bool
+	 */
 	public function exists() {
 		return file_exists($this->logFile);
 	}
 
+	/**
+	 * Return the content of the log file.
+	 * @return string
+	 */
 	public function content() {
-		return file_get_contents($this->logFile);
+		return $this->exists() ? file_get_contents($this->logFile) : '';
```

Comment:
Can we reply that this log doesn't exists? otherwise it might look like a bug and someone needs to log into the server and check if it's there or something else has crashed..